previewer/main.js

```diff
@@ -35,13 +35,19 @@ function initApp() {
   console.log("Using hatch folder: " + hatchFolder);
 
   assetMap = new Map()
+  tagsMap = new Map()
 
   let hatchLanguage, hatchName;
   try {
       const manifest = loadManifest(hatchFolder);
       manifest.assets.forEach(function(asset) {
           const id = asset.asset_id;
           assetMap.set(id, loadMetadata(hatchFolder, id))
+          assetMap.get(id).tags.forEach(tag => {
+              if (tagsMap.has(tag))
+                  tagsMap.set(tag, tagsMap.get(tag) + 1)
+              else tagsMap.set(tag, 1)
+          });
```

Comment:
I _think_ this can be done more efficiently with `map` but can't quite put my finger on the implementation off the top of my head but regardless, you can do (untested):
```
tagsMap = {}

assetMap.get(id).tags.forEach(tag => {
    tagsMap[tag] = (tagsMap[tag] || 0) + 1;
});
```